### DISCUSSION TITLE
Expedition 6 - The Blighted- Triggering the Freighter battle with Worlds part 1 ( phase 1, #3)

### DISCUSSION THREAD

**Discussion by KurtisVargas** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#20] [General]
Has anybody been able to trigger the Freighter battle (phase 1, number 3) in Expedition 6 in a recent Worlds versions?   According to guides it should trigger on the first jump, but I've tried 3 playthroughs including a 100% vanilla game and even waiting >3 hours, 20+ warps, can't get it to trigger. 

Any suggestions or ideas?

**Comment by cwmonkey** [comment]
Confirming the issue. I just did my first jump and there is no freighter battle. My guess is there is either a setting or code to make that first battle happen. Hopefully it's a setting so I can just turn it off, because if it's code there may be no fix for this aside from replacing the milestone.

Worth noting I just did Leviathan and I was absolutely inundated with freighter rescues and freighter battles.

**Reply by cwmonkey** [reply]
Looks like there is indeed a setting: `"FreighterBattleEarlyWarpsOverride": 1,`

For expeditions after this they have it set to `"FreighterBattleEarlyWarpsOverride": -1,`

This could mean I need to set it to `-1` so the freighter battle will trigger normally on the 3rd (I think) jump.

Before I test this I will warp 4 more times (for a total of 5 warps) to make sure the freighter rescue battle isn't happening for me.

**Reply by cwmonkey** [reply]
After doing 6 warps (last one to the rdv1 system) I set `"FreighterBattleEarlyWarpsOverride": -1,` and restarted the game. 5 warps later still no freighter rescue. This either means the fix won't work, or it just won't work on in-progress expeditions.

Gonna try with a new expedition now.

**Reply by cwmonkey** [reply]
Alright, giving up on this approach. Simply going to make this milestone "blow up 40 asteroids"

**Reply by KurtisVargas** [reply]
Thank you!  It's been absolute blast playing these old expeditions and greatly appreciate all the work you and leodium have done to make it work.

**Reply by cwmonkey** [reply]
@KurtisVargas np! Glad to fix expedition breaking bugs like this.

My workaround is available here: https://cwmonkey.github.io/nms-expeditions/

I'll update this repo momentarily.

**Reply by cwmonkey** [reply]
And updated in this repo in 25ea256

**Reply by KurtisVargas** [reply]
Confirmed working for all three save files!
